### PR TITLE
Add home split options

### PR DESCRIPTION
**Pull request by beschoenen**
Add split options to the home screen.

image

image

Current options are: None, Anime, Sports, Anime & Sports, Root dir.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by beschoenen** [comment]
@miigotu Should this new setting be saved in the `config.ini`?

**Comment by miigotu** [comment]
Cant just select one root dir anymore? Point of that is the shows for one user (wife/kids/etc) can be put in their own root and use that option to filter which user's shows are displayed.

**Comment by beschoenen** [comment]
image

image

**Comment by beschoenen** [comment]
@miigotu I think I'm missing something, `home_view` doesn't get saved to the `config.ini`.

File: sickbeard/__init__.py [hunk 1]
```diff
@@ -1301,6 +1299,7 @@ def path_leaf(path):
         METADATA_MEDE8ER = check_setting_str(CFG, 'General', 'metadata_mede8er', '0|0|0|0|0|0|0|0|0|0')
 
         HOME_LAYOUT = check_setting_str(CFG, 'GUI', 'home_layout', 'poster')
+        HOME_VIEW = check_setting_int(CFG, 'GUI', 'home_view', None)
```
Comment (miigotu) [inline sickbeard/__init__.py]: Its not an int, so do:
```python
HOME_VIEW = check_setting_str(CFG, 'GUI', 'home_view', None)
```

Comment (miigotu) [reply sickbeard/__init__.py]: To fix saving to config, around line 2236 add 
```python
'home_view': HOME_VIEW
```

Comment (miigotu) [inline sickbeard/__init__.py]: Er, Line 2236 -ish, it wont let me comment there

**Review by miigotu — CHANGES_REQUESTED** [review]

**Comment by miigotu** [comment]
Maybe needs a way to toggle or collapse individual roots? I like being able to select just one root dir to view at a time with 2 clicks atm

**Comment by beschoenen** [comment]
Maybe a little arrow before the path of a root directory? (so only when split by root directory)

**Comment by miigotu** [comment]
maybe. I think you could also leave the  drop down how I have it atm and have that collapse or hide all other roots so only one is showing. Another cool option would be to have root dir "groups" that you can label what you want and add a few root dirs to.

Like
`Kids` = `/RAID1/Videos/Kids_Shows`, `/RAID1/Videos/Christmas`
`Mommy` = `/RAID1/Videos/Shows`, `/RAID1/Videos/Christmas`

Ofc it would be handled by a string of numbers:
```
[General]
    ROOT_DIRS = 1|/RAID1/Videos/Kids_Shows,2|/RAID1/Videos/Christmas,3|/RAID1/Videos/Shows,4|/RAID1/Videos/Shows/Anime,5|/RAID1/Videos/Documentaries
[Root_Groups]
    Mommy = 23
    Kids = 12
    Dad = 345
```

That would allow for a pretty/customizable name for root dirs and/or in a dropdown or toggles like in the show page
image


That would get us started down the road of being able to split up SR by user I think

**Comment by beschoenen** [comment]
That's a nice idea, but wouldn't it be better to do that in another PR?

**Pull request closed without merging**